src/sentry/middleware/integrations/integration_control.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import logging
+import re
+from typing import Mapping, Type
+
+from sentry.silo import SiloMode
+from sentry.types.integrations import EXTERNAL_PROVIDERS, ExternalProviders
+
+from .parsers import SlackRequestParser
+from .parsers.base import BaseRequestParser
+
+logger = logging.getLogger(__name__)
+
+
+class IntegrationControlMiddleware:
```

Comment:
I think I'll use this name but change it to `IntegrationRequestRegionForwarding` since they're not always conventional webhooks (e.g. Slack requires a response)

Reply:
Actually, on second thought, the requests might also be consumed and responded to directly from the control, so it might not be accurate to change anything just yet